README.md

```diff
@@ -1,61 +1,42 @@
-# grunt-standard
+# grunt-standard [![JavaScript Standard Style](https://img.shields.io/badge/code%20style-standard-brightgreen.svg)](http://standardjs.com/) [![GitHub release](https://img.shields.io/github/release/pdehaan/grunt-standard.svg)](https://github.com/pdehaan/grunt-standard/releases) [![npm (scoped)](https://img.shields.io/npm/v/@pdehaan/grunt-standard.svg)](https://www.npmjs.com/package/@pdehaan/grunt-standard) [![license](https://img.shields.io/github/license/pdehaan/grunt-standard.svg)](https://github.com/pdehaan/grunt-standard/blob/master/LICENSE) [![Twitter Follow](https://img.shields.io/twitter/follow/pdehaan.svg?style=social&label=Follow)](https://twitter.com/pdehaan) [![GitHub followers](https://img.shields.io/github/followers/pdehaan.svg?style=social&label=Follow)](https://github.com/pdehaan)
```

Comment:
You'll want to move the GitHub and Twitter and npm info to your name -- also, i think the npm one is wrong anyways since it wasnt a scoped package).
(I already added you as an npm collaborator -- feel free to remove me)